lib/src/configuration.dart

```diff
@@ -85,7 +92,7 @@ class Configuration {
     // otherwise symbolication fails, the default path for the web build folder is build/web
     // but can be customized so making it flexible.
     final webBuildPath = config?['web_build_path']?.toString() ?? 'build/web';
-    webBuildFilesFolder = '$buildFilesFolder$_fileSeparator$webBuildPath';
+    webBuildFilesFolder = '$buildFilesFolder/$webBuildPath';
```

Comment:
IIRC, this is necessary for supporting different platforms.

Reply:
windows doesn't care about forward slashes - works fine as well.

also see previous line: `build/web` - so apparently we already weren't consistent

Reply:
On Java it does, we had issues with the Gradle plugin because of that, and using `File.separator` fixed the issue.
I'd rather fix `build/web` then, apparently an oversight.
So I'd rather prefer this than sorry.

Reply:
https://github.com/getsentry/sentry-dart-plugin/pull/52/commits/7ccd117568a094c3d2993b0a4b94eb1100cd62f2